### ISSUE TITLE
Transfer Bitcoin Unknown

### ISSUE THREAD

**Issue by htfupeach** [spesmilo/electrum#2646]
i am freaking out right now, i transferred my bitcoin to electrum thinking i knew what i was doing but apparently i didn't i have no idea what i am looking at and where my bitcoin has gone. i am really confused and have no idea where do start. i'm new to bitcoin i am trying to learn everything but when there is so many people scamming i have no idea what to believe anymore. except forums. im sorry if this is vague but that is my knowledge of what i'm doing which doesn't help at all.. =O  

basically, i need help to track my money that i transferred from coinbase.com to electrum.
